src/write.c

```diff
@@ -477,7 +477,7 @@ static void avifEncoderBackupSettings(avifEncoder * encoder)
 
 // This function detects changes made on avifEncoder. It returns true on success (i.e., if every
 // change is valid), or false on failure (i.e., if any setting that can't change was changed). It
-// reports detected changes in encoderChanges.
+// reports detected changes in encoderChanges, or sets it to 0 if none.
```

Comment:
Nit/optional: Here it is clear that 0 means no detected changes. It would be more important to document that `encoderChanges` is a bitwise-OR of detected changes.